vllm/config.py

```diff
@@ -238,21 +239,21 @@ def _sparsity_structure_from_config(
         # check for valid dtype
         if dtype not in supported_sparsity_dtypes:
             logger.warning(
-                "Sparsity is only supported for float16 and bfloat16 "
+                f"Sparsity is only supported for {supported_sparsity_dtypes}"
                 "dtypes. Running the models without sparse kernels.")
```

Comment:
I actually meant the current dtype, but supported dtypes are good too!
```suggestion
                f"dtypes, not {dtype}. Running the models without sparse kernels.")
```